test/clt-tests/buddy/test-fuzzy-autocomplete-http.rec

```diff
@@ -9,8 +9,9 @@ apt-get install jq -y > /dev/null; echo $?
 debconf: delaying package configuration, since apt-utils is not installed
 0
 ––– input –––
-php -d memory_limit=-1 ./test/clt-tests/scripts/load_us_names_min_infix_len.php 1000 10 1000000 5 2 > /dev/null
+php -d memory_limit=2G ./test/clt-tests/scripts/load_names_attr.php 100000 4 1000000 1 2G 1 2 > /dev/null; echo $?
```

Comment:
Batch size is 100K vs 1K previously. Is it on purpose? 100K seems to be too high, but if it's done intentionally - ok.

Reply:
The multiplier was 5, now it's 1. So previously there was 5M docs in the table, now only 1M. Is it on purpose?

Reply:
Fixed